jobbergate-test/jobbergate_test/main.py

```diff
@@ -11,19 +17,38 @@ def validate_access_token():
 
     logger.info("Start validating access token")
 
-    with handle_errors("Please, login in to Jobbergate. Failed to validate access token."):
+    with handle_errors(
+        "Please, login in to Jobbergate. Failed to validate access token."
```

Comment:
I think the ordering of the message is backwards. To me, it would make more sense to have it "Failed to validate access token. Please login to Jobbergate to continue." Or something like that.

Reply:
solved in [89b7185](https://github.com/omnivector-solutions/jobbergate/pull/225/commits/89b718599a465dcee2e518058ddb3dd827474ee4)